tests/test_helpers.py

```diff
@@ -45,9 +45,23 @@
     (["helper", "l_contacts", "run", "-n", "ny", "-i", "col", "-o", "coffee", "-d", "2019-01-15", "-r", "2"],
      ""
      ),
+    # (["helper", "u_milestone", "20sb_firstprojectum", "--number", "5",
+    #   "--due_date", "2020-05-10", "--status", "finished"],
+    #  "20sb_firstprojectum has been updated in projecta\n"
+    #  ),
+    (["helper", "u_milestone", "20sb_firstprojectum"],
+     "Please choose from one of the following to update/add:\n"
+     "1. name: deliverable     due date:2021-05-05\n"
```

Comment:
I think it would be better if these were ordered on chronological order by due date, and also showed the status (e.g. finished, proposed, converged

Reply:
done!

Reply:
done!